tests/test_samples.py

```diff
@@ -1230,3 +1230,8 @@ def test_constructors():
     assert samples.T[0].islabelled()
 
     assert isinstance(samples['x0'].to_frame(), WeightedLabelledDataFrame)
+
+
+def test_old_gui():
+    with pytest.raises(TypeError):
+        MCMCSamples(root='./tests/example_data/pc')
```

Comment:
Use `Samples` instead of `MCMCSamples` here? That should cover both `MCMCSamples` and `NestedSamples`, too.

Reply:
Testing for all three now.